Scripts/Modules/HierarchyModule.cs

```diff
@@ -28,10 +28,12 @@ void Awake()
 				m_IgnoreList.Add(manager.gameObject);
 			}
 
+#if UNITY_EDITORVR
```

Comment:
Actually I added this because the compilation test failed. Was probably introduced by my hierarchy update from a few weeks ago.